observability/akka-http-metrics/src/main/scala/com/daml/metrics/akkahttp/WebSocketMetrics.scala

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics.akkahttp
+
+import com.google.common.base.Utf8
+
+import akka.util.ByteString
+import akka.stream.scaladsl.{Flow, Sink}
+import akka.http.scaladsl.model.ws.{Message, TextMessage, BinaryMessage}
+
+import com.daml.metrics.api.MetricsContext
+import com.daml.metrics.api.MetricHandle.Counter
+
+/** Support to capture metrics on websockets on akka http
+  */
+object WebSocketMetrics {
+
+  /** Wraps the given flow, to capture in the given metrics, the following signals:
+    * - total number of received messages
+    * - size of the received messages
+    * - total number of sent messages
+    * - size of the sent messages
+    */
+  def withGoldenSignalsMetrics[M](
+      wsReceivedTotal: Counter,
```

Comment:
This method is within a companion object concerning websocket specific metrics, hence, I'd suggest removing the `ws` prefix from these argument names to improve readability.